src/_includes/layout.njk

```diff
@@ -147,22 +147,22 @@
           <div class="col-md-6">
             <ul class="list-inline social-buttons">
               <li class="list-inline-item">
-                <a href="https://www.facebook.com/codeconnector" target="blank">
+                <a href="https://www.facebook.com/codeconnector" target="blank" aria-label="Code Connector's Facebook navigation link">
```

Comment:
Screen readers will announce that this is a link, so a better label would just be "Code Connector's Facebok" or "Facebook"

Reply:
Fixed